apps/glusterfs/app_volume.go

```diff
@@ -95,6 +95,18 @@ func (a *App) VolumeCreate(w http.ResponseWriter, r *http.Request) {
 		}
 	}
 
+	for _, volumeOption := range msg.Options {
+		switch volumeOption {
+		case "encryption":
```

Comment:
@humblec, yes initially I felt we should do wrapping with higher-level heketi options, but then, with arguments by you and @jarrpa, and a recent real-world case that I ran across, I was slowly becoming convinced that a verbatim pass-through might be the best option.

And your current patch is not a simple whitelist, in the sense that it has a list of options which to pass through verbatim, but it is a higher level wrapping of some allowed combination of options (currently only the encryption ones). So it's more complicated than a simple whiltelist (adding convenience for the user but taking flexibility).

In the previous comment, I tried to list the pros and cons of the two approaches, and gave my preference, based on this newer perspective. I don't think anymore that we should have that additional intelligence in heketi. But I would like to hear other people's opinions before we change code again... @jarrpa implicitly voted for the simple pass-through - is that still valid? @humblec what do you think according to the analysis above? @lpabon @raghavendra-talur @MohamedAshiqrh .... any opinions?